hazelcast/src/test/java/com/hazelcast/client/impl/MemberImplTest.java

```diff
@@ -41,15 +42,15 @@ public void testConstructor() {
 
     @Test
     public void testConstructor_withAddress() {
-        MemberImpl member = new MemberImpl(address);
+        MemberImpl member = new MemberImpl(address, Version.of("3.8.0"));
```

Comment:
Is this not going to cause problems as soon as we move to the development of 3.9? So do we really want to have a hardcoded 3.8.0 in the system?